DXMainClient/Online/Connection.cs

```diff
@@ -324,6 +326,110 @@ private void HandleComm(object client)
             }
         }
 
+        /// <summary>
+        /// Get all IP addresses of Lobby servers by resolving the hostname and test the latency to the servers.
+        /// The maximum latency is defined in <c>MAXIMUM_LATENCY</c>, see <see cref="Connection.MAXIMUM_LATENCY"/>.
+        /// </summary>
+        /// <returns>A list of available Lobby servers sorted by latency.</returns>
+        private IEnumerable<Server> GetAvailableServerList()
+        {
+            var availableServerAndLatencyDict = new Dictionary<Server, long>();
+
+            try
+            {
+                var dnsAndPingTasks = new List<Task>();
+
+                foreach (var server in Servers)
+                {
+                    var serverHostnameOrIPAddress = server.Host;
+                    var serverName = server.Name;
+                    var serverPorts = server.Ports;
```

Comment:
Try to avoid using `var` on every ocassion, it's clashing with client code style.

Reply:
Fixed in c09827cbb9556f93fc5acf67448ece2194546faa .